webui/react/src/pages/TrialDetails/Profiles/Charts/TimingMetricChart.tsx

```diff
@@ -17,10 +18,11 @@ export const TimingMetricChart: React.FC<ChartProps> = ({ trial, getOptionsForMe
 
   return (
     <Section bodyBorder bodyNoPadding title="Timing Metrics">
-      <UPlotChart
-        data={timingMetrics.data}
-        noDataMessage="No data found. Timing metrics may not be available for your framework."
-        options={options}
+      <LineChart
+        series={timingMetrics.data}
+        xAxis={XAxisDomain.Time}
+        xLabel="Time"
+        yLabel={options.axes?.[1].label ?? ''}
```

Comment:
since we are ignoring the rest of what `getOptions` returns, can we change that to just be a `getAxis`?